### DISCUSSION TITLE
Are you aware of them AWDark

### DISCUSSION THREAD

**Discussion by oehhar** [rdbende/Azure-ttk-theme#27] [General]
Dear RDBende,

thank you for your great themes for ttk, that is awesome.

Are you aware of the themes created by Brad Lanam: AWThemes ?

https://wiki.tcl-lang.org/page/awthemes

There are some great ideas, like scaling and color configuration.

Unfortunately, Brad left the Tk community and the themes are not maintained any more.

I am persoally using AWDark with AndroWish on Android. The scaling property is required there, as you have quite varying screen resolutions on Android.

By chance, is scaling already supported by your themes ?

Thank you, sorry for the dump question without checking it out on the practical way.
But Christian Werner (Mr Androwish) may include your themes in the Androwish distribution.

Take care,
Harald

**Comment by rdbende** [comment]
Hello Oehhar, thanks for checking out my theme!

I've used awthemes, and I think it has really nice features, but I've never looked at it's code. Unfortunately scaling is not supported by my themes at the moment, as you can see in rdbende/Sun-Valley-ttk-theme#18, although I have an [svg version of Sun-Valley](https://github.com/rdbende/Sun-Valley-ttk-theme-svg).

I want to implement some kind of scaling and accent color features for my themes, but I don't plan to work on that in the near future, however I do plan to make some kind of built-in theme-scaling and accent color capabilities for [tukaan](https://github.com/tukaan/tukaan), and it's possible that my themes will then get something like that.

**Comment by oehhar** [comment]
Thank you for the comment. The community and myself are excited by your work.
Thanks,
Harald

**Comment by oehhar** [comment]
By the way, there was a discussion about your themes on the comp.lang.tcl news list.
Some people made tests and reported the caveat that rendering is very slow on WIndows. Brad Lanam has worked a lot on this subject and minimized the used image size.
This is due to ineffective solutions in Tk and it would be better to fix it in Tk.

Thanks again,
Harald
